### DISCUSSION TITLE
Most probably a bricked tx and don't know how to recover

### DISCUSSION THREAD

**Discussion by ghost** [OpenI6X/opentx#421] [General]
My tx turned off due to low battery voltage and now it doesn't turn on at all. When the power switch is on it draws around 50ma at 5v. I have tried to reflash the firmware but it was not detected by the pc. Also tried to put it in the dfu by r53 pad but nothing happened.  

Can anyone please help me?

**Comment by ajjjjjjjj** [comment]
It is impossible to break DFU mode unless there is hardware damage. Check voltages of linear regulators outputs.

**Reply by ghost** [reply]
I have checked the voltage regulators and both of them have 3.3v output. 
I have tried everything from the internet to solve the issue without success. The board draws 54ma at 6v when all the switches and gimbals are disconnected which turns to 0ma after I removed the a7105.
I am also a bit confused. I didn't find any physical damage on the board.
